pkg/antctl/raw/set/flow-aggregator/command_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flow_aggregator
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	testclient "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestSetBoolOrFail(t *testing.T) {
+	var b bool
+	var value string
+	value = "1"
+	err := setBoolOrFail(&b, value)
+	assert.NoError(t, err)
+	value = "tt"
+	err = setBoolOrFail(&b, value)
+	assert.Error(t, err)
+}
+
+func TestSetStringOrFail(t *testing.T) {
+	var b, value string
+	value = "11mm"
+	_ = setStringOrFail(&b, value)
+	assert.Equal(t, b, value)
+}
+
+func TestSetCommitIntervalOrFail(t *testing.T) {
+	var b, value string
+	value = "1m25s"
+	err := setCommitIntervalOrFail(&b, value)
+	assert.NoError(t, err)
+	value = "0.5s"
+	err = setCommitIntervalOrFail(&b, value)
+	assert.Error(t, err)
+	value = "0.5ss"
+	err = setCommitIntervalOrFail(&b, value)
+	assert.Error(t, err)
+}
+
+func TestGetFAConfigMap(t *testing.T) {
+	k8sClient := testclient.NewSimpleClientset()
+	faConfigMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testConfigMap",
+			Namespace: "flow-aggregator",
+		},
+	}
+	_, err := k8sClient.CoreV1().ConfigMaps("flow-aggregator").Create(context.TODO(), faConfigMap, metav1.CreateOptions{})
+	if err != nil {
+		fmt.Println(err)
+	}
+	_, err = GetFAConfigMap(k8sClient, "testConfigMap")
+	assert.NoError(t, err)
+	result, _ := GetFAConfigMap(k8sClient, "testConfigMap1")
```

Comment:
You need to verify the expected error when `GetFAConfigMap` failed to get configMap, eg: `configMapName` length is zero, `testConfigMap1` not found error.
I feel it's better to refer to other commands to add test for `updateRunE`.

Reply:
sure